src/main.rs

```diff
@@ -15,12 +15,19 @@ struct Args {
     break_time: u64,
     #[arg(short = 'i', long = "hide-image", default_value = "false")]
     hide_image: bool,
+    #[arg(short = 's', long = "sound")]
+    sound: Option<String>,
+    #[arg(short = 'n', long = "no-sound", help="default to false")]
+    no_sound: Option<bool>
```

Comment:
Important: since this argument is a boolean, we can make the user experience easier by making it required, but setting it false by default. 
```suggestion
    #[arg(short = 'n', long = "no-sound", help="default to false", default_value = "false")]
    no_sound: bool
```
That way, if the user wants to remove the sound, she/he can just do `pomodoro -n`. Moreover, we then don't need to call `unwrap_or_default()` on this variable.